### PR TITLE
Feat/pm 1063/catalog carousel

### PR DESCRIPTION
**Pull request by mati-1**
[PM-1063](https://playmakerpro.atlassian.net/browse/PM-1063)

- added simple carousel (without item variants)
- corrected all styles from figma at register pages

### CODE REVIEW

File: components/login-register/common/Input/checkbox.tsx [hunk 1]
```diff
@@ -1,46 +1,72 @@
...
+        <div className="w-full pt-5">
+          <Input
+            title="Czym się zajmujesz?"
+            inputValue={otherObligationsValue || ''}
+            type="text"
+            errorMessage={errors.otherObligationsValue?.message}
+            {...register('otherObligationsValue')}
+          />
+        </div>
+      )}
```
Comment (dejwi) [inline components/login-register/common/Input/checkbox.tsx:69]: I don't think it should be here, maybe pass a `children` prop and do stuff there

File: components/login-register/register/pages/profile-type-page/profile-type-page.tsx [hunk 2]
```diff
@@ -10,35 +11,79 @@ import { InnerFormWrapper } from '@/components/login-register/common/FormWrapper
...
+    setValue,
+    setError,
+    formState: { errors },
+  } = useForm<IProfileTypeInput>({
+    resolver: yupResolver(profileTypeFormSchema),
+    mode: 'onSubmit',
+    reValidateMode: 'onChange',
+    defaultValues: {
+      profileType: ctxProfileType,
+      otherObligationsValue: customProfileType,
```
Comment (dejwi) [inline components/login-register/register/pages/profile-type-page/profile-type-page.tsx:39]: maybe something like this ?
```suggestion
      customProfileTypeName: customProfileType,
```

**Review by dejwi — CHANGES_REQUESTED** [review]
image

odkrywaj and wiecej are not on the same level

on mobile height of card is too big and dots should be hidden

image

it isn't responsive

and maybe choose a different image for this ? idk where you've found this one but it looks stretched XD

File: components/login-register/register/pages/profile-type-page/profile-type-page.tsx (hunk 2 above)
Comment (mati-1) [reply components/login-register/register/pages/profile-type-page/profile-type-page.tsx:39]: XDD

**Comment by mati-1** [comment]
when pagination is extended, layout extends beyond the screen

image

**Review by dejwi — CHANGES_REQUESTED** [review]
BRO PLEASE FOLLOW FIGMA STYLES CAREFULLY !!😭

In next pr's please don't mix stuff, carousel and register stuff should be in separate pr's

Maybe I missed some things so please double check

--

(figma)
image

(our)
image

--

dots should be higher check figma
image

--

it should be further from footer(mobile)
image

**Comment by mati-1** [comment]
> (figma) image
> 

shadow at the right side is only for mobile devices? I didnt create it cos this component will be a little different in the future (kamil said this)

**Comment by mati-1** [comment]
and you re right bout register stuff. I wasnt thought like that xD

**Comment by mati-1** [comment]
i changed the footer padding in the teams catalog branch, dont look at it there

File: components/catalogs/players/carousel/carousel-section.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Heading } from '@/components/profile/common/Heading'
+import { SectionContainer } from '@/components/profile/common/SectionContainer'
+import ChevronIcon from '@/icons/ChevronRight.svg'
+
+import { DiscoverCarousel } from '../../common/CatalogCarousel/discover-carousel'
+
+export const CarouselSection = () => (
```
Comment (dejwi) [inline components/catalogs/players/carousel/carousel-section.tsx:7]:
```suggestion
export const DiscoverCarouselSection = () => (
```

File: components/catalogs/players/carousel/carousel-section.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        type="button"
+        className="flex items-center gap-2 text-base text-label-thin-md sd:text-label-thin-lg"
+      >
+        Więcej <ChevronIcon className="fill-base icon-16 sd:icon-20" />
+      </button>
+    </div>
+
+    <DiscoverCarousel />
+  </SectionContainer>
+)
```
Comment (dejwi) [inline components/catalogs/players/carousel/carousel-section.tsx:23]: it's reused in clubs catalog as well so better place it in `catalogs/common/DiscoverCarousel` (use and rename already existing folder there ~`CatalogCarousel` )

**Review by dejwi — CHANGES_REQUESTED** [review]
I didn't mean the shadow, items look different
image
vs
image

and on mobile this section should be further from pagination

**Review by dejwi — APPROVED** [review]
🎉

**Pull request merged**